accessibility.html

```diff
@@ -0,0 +1,281 @@
+                    &lt;form id="slack-integration"&gt;
+                      &lt;input id="mail-for-slack" type="email" placeholder="me@mydomain.com" autofocus="true" required/&gt;
+                      &lt;input type="submit" id="submit-slack" value="Get your Invite"/&gt;
+                      &lt;div class="message"&gt;&lt;/div&gt;
+                    &lt;/form&gt;
+                  </code>
+                </pre>
+              </p>
+              <p>
+                The following PR suggests some improvements that can be made to the form to make it more accessible: 
+                <a href="https://github.com/hoodiehq/hood.ie/pull/188">https://github.com/hoodiehq/hood.ie/pull/188</a>
+              </p>
+              <p>
+                <ul>
+                  <li>
+                    We can attach label to the input box so the user knows what they should be entering in the box. 
+                    Placeholder text may not be read out, nor is it supported by older browsers.
+                  </li>
+                  <li>
+                    We can make sure the inputs are not autofocused, because a screenreader will lose all contextual information 
+                    if they are forced to jump to content in the middle of a page when they visit it.
+                  </li>
+                  <li>
+                    The button can be changed to be an actual semantic <em>&lt;button&gt;</em> rather than an <em>&lt;input&gt;</em> with its type set as button.
+                  </li>
+                  <li>
+                    Strongly coloured warning and success messages can be added to show the status of the form submission
+                  </li>
+                </ul>
+              </p>
+              <p>
+                <pre>
+                  <code class="language-html">
+                    &lt;form id="slack-integration"&gt;
+                      &lt;label for="mail-for-slack">Enter your email address&lt;/label&gt;
+                      &lt;input id="mail-for-slack" type="email" placeholder="me@mydomain.com" required>
+                      &lt;button type="submit" id="submit-slack">Get your invite&lt;/button&gt;
+                      &lt;div id="slack-form-message" class="message"&gt;&lt;/div&gt;
+                    &lt;/form&gt;
+                  </code>
+                </pre>
+              </p>
+              <img src="/src/img/accessibility-slack-after.png" alt="">
+              
+            </article> 
+          </section>
+          
+          <section class="cb">
+            <article id="tools" role="article">
+              <h2>Accessibility Testing</h2>
+              <p>
+                <ul>
+                  <li>
+                    Screenreaders like <a href="https://www.apple.com/uk/accessibility/osx/voiceover/">VoiceOver</a> 
+                    can help us to understand if our content makes sense and is structured well
+                  </li>
```

Comment:
I thought that NVDA was not free at the time of writing this, so I am really happy that I am wrong about that and will update!